core/app/models/mno_enterprise/impac/dashboard.rb

```diff
@@ -22,13 +22,13 @@ def full_name
     end
 
     # Return all the organizations linked to this dashboard and to which the user has access
-    # If the dashboard is a template, return all the current user's organization
+    # If the dashboard is a template, return all the current user's organizations
     def organizations(org_list = nil)
       if org_list
         return org_list if dashboard_type == 'template'
-        org_list.to_a.select { |e| self.organization_ids.include?(e.uid) }
+        org_list.to_a.select { |e| organization_ids.include?(e.uid) || organization_ids.include?(e.id) }
```

Comment:
Isn't in only `uids` (based on the `where` clause below)?

Reply:
no: ids are passed when the request is a `create` - this organizations uids/ids thing on the impac components are super confusing and should be refactored accross all the applications